src/app/icd/ICDManager.cpp

```diff
@@ -167,6 +170,8 @@ void ICDManager::UpdateOperationState(OperationalState state)
         {
             uint16_t activeModeThreshold = IcdManagementServer::GetInstance().GetActiveModeThreshold();
             DeviceLayer::SystemLayer().ExtendTimerTo(System::Clock::Timeout(activeModeThreshold), OnActiveModeDone, this);
+            DeviceLayer::SystemLayer().ExtendTimerTo(System::Clock::Timeout(activeModeThreshold - ICD_ACTIVE_TIME_JITTER_MS),
```

Comment:
Same same timer or two timer ?
What is ** OnActiveModeAlmostDone ? 
i don't feel this is a good naming...

Reply:
I am not too fond of the OnActiveModeAlmostDone either, how about, OnActiveModeJitter, or OnEnterTransitionToIddle?

Reply:
Changed to OnTransitionToIdle